### PR TITLE
Finish renaming price feeds to data feeds.

### LINKED ISSUE #295: Consider renaming "Price Feeds" pages
There are now more than just price feeds on the price feeds pages. We should consider renaming them to something like "data feeds" to match data.chain.link. Additionally, we should consider separating out price feeds and other types of data feeds.

### PR DESCRIPTION
**Pull request by dwightjl**
Fixes #295

### CODE REVIEW

File: docs/Using Price Feeds/get-the-latest-price-solana.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ This guide requires the following tools:
   - [Rust](https://www.rust-lang.org/tools/install)
   - [The Solana CLI](https://docs.solana.com/cli/install-solana-cli-tools#use-solanas-install-tool)
   - [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- Call the deployed program to read from the [SOL / USD price data feed](https://explorer.solana.com/address/FmAmfoyPXiA8Vhhe6MZTr3U6rZfEZ1ctEHay1ysqCqcf?cluster=devnet):
+- Call the deployed program to read from the [data feed for SOL / USD prices](https://explorer.solana.com/address/FmAmfoyPXiA8Vhhe6MZTr3U6rZfEZ1ctEHay1ysqCqcf?cluster=devnet):
   - [Node.js 12 or higher](https://nodejs.org/en/download/)
```
Comment (dmitrydao) [inline docs/Using Price Feeds/get-the-latest-price-solana.md:49]: I think this should not be plural as it's one price - > data feed for SOL / USD price

File: docs/Using Price Feeds/get-the-latest-price-solana.md [hunk 2]
```diff
@@ -170,7 +170,7 @@ The example code operates using the following process:
...
 
-1. The `process_instruction` function in `lib.rs` receives the transaction with the specified feed address and handles the steps to retrieve and store the price data on-chain. The function also calls `get_price()` from the [chainlink-solana](https://github.com/smartcontractkit/chainlink-solana) package, which gets imported from GitHub in the `Cargo.toml` file. See the [`Cargo.toml`](https://github.com/smartcontractkit/chainlink-solana-demo/blob/main/Cargo.toml#L19) file, which maps the `chainlink-solana` package to `chainlink`.
+1. The `process_instruction` function in `lib.rs` receives the transaction with the specified feed address and handles the steps to retrieve and store the data on-chain. The function also calls `get_price()` from the [chainlink-solana](https://github.com/smartcontractkit/chainlink-solana) package, which gets imported from GitHub in the `Cargo.toml` file. See the [`Cargo.toml`](https://github.com/smartcontractkit/chainlink-solana-demo/blob/main/Cargo.toml#L19) file, which maps the `chainlink-solana` package to `chainlink`.
```
Comment (StephenFluin) [inline docs/Using Price Feeds/get-the-latest-price-solana.md:173]: This one is explicitly price data (see `get_price`), so we could consider leaving it, but I think it's okay to change anyway.

File: docs/Using Price Feeds/get-the-latest-price-solana.md [hunk 3]
```diff
@@ -16,24 +16,24 @@ Price data feeds are available on the following networks:
...
 
-To get the full list of Chainlink Price Feeds running on the Solana Devnet, see the [Solana Feeds](/docs/solana-price-feeds/) page.
+To get the full list of Chainlink Data Feeds running on the Solana Devnet, see the [Solana Feeds](/docs/solana-price-feeds/) page.
 
 You can view the program ID that owns these feeds in the [Solana Devnet Explorer](https://explorer.solana.com/address/2yqG9bzKHD59MxD9q7ExLvnDhNycB3wkvKXFQSpBoiaE?cluster=devnet).
 
 > 📘 Note
 >
-> This guide shows you how to deploy and retrieve price data using the Solana network. If you need to use EVM-compatible networks, see [Consuming Price Feeds (EVM)](../get-the-latest-price/).
+> This guide shows you how to deploy and retrieve data using the Solana network. If you need to use EVM-compatible networks, see [Consuming Data Feeds (EVM)](../get-the-latest-price/).
```
Comment (StephenFluin) [inline docs/Using Price Feeds/get-the-latest-price-solana.md:29]: Should we fix this URL and forward?

File: docs/Using Price Feeds/get-the-latest-price-solana.md [hunk 4]
```diff
@@ -2,12 +2,12 @@
 layout: nodes.liquid
 section: smartContract
 date: Last Modified
-title: "Consuming Price Feeds (Solana)"
+title: "Consuming Data Feeds (Solana)"
 permalink: "docs/get-the-latest-price-solana/"
-whatsnext: {"Historical Price Data":"/docs/historical-price-data/", "Price Feeds API Reference":"/docs/price-feeds-api-reference/", "Price Feeds Contract Addresses":"/docs/reference-contracts/"}
+whatsnext: {"Historical Price Data":"/docs/historical-price-data/", "Data Feeds API Reference":"/docs/price-feeds-api-reference/", "Data Feed Contract Addresses":"/docs/reference-contracts/"}
```
Comment (StephenFluin) [inline docs/Using Price Feeds/get-the-latest-price-solana.md:7]: Should we fix these URLs?

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Left a couple thoughts/ideas, but we should definitely fix the typo in the commit message :).

File: docs/Using Price Feeds/get-the-latest-price-solana.md (hunk 1 above)
Comment (dwightjl) [reply docs/Using Price Feeds/get-the-latest-price-solana.md:49]: Updated. We can make this part of our style guide for how to refer to data feeds.

File: docs/Using Price Feeds/get-the-latest-price-solana.md (hunk 2 above)
Comment (dwightjl) [reply docs/Using Price Feeds/get-the-latest-price-solana.md:173]: I was 50/50 on making this change as well. I'll revert it.

File: docs/Using Price Feeds/get-the-latest-price-solana.md (hunk 3 above)
Comment (dwightjl) [reply docs/Using Price Feeds/get-the-latest-price-solana.md:29]: We will handle URL changes and redirects in a separate change. There are several.

File: docs/Using Price Feeds/get-the-latest-price-solana.md (hunk 4 above)
Comment (dwightjl) [reply docs/Using Price Feeds/get-the-latest-price-solana.md:7]: We will handle URL changes and redirects in a separate change. There are several.

**Review by dmitrydao — APPROVED** [review]

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**